.buildkite/heartbeat/heartbeat-pipeline.yml

```diff
@@ -26,6 +26,33 @@ env:
   TEST_COVERAGE: "true"
 
 steps:
+  - group: "Heartbeat Check/Update"
+    key: "heartbeat-check-update"
+    
+    steps:
+      - label: "Run check/update"
+        command: |
+          make -C heartbeat check update
```

Comment:
same question as in https://github.com/elastic/beats/pull/39870#discussion_r1642372419

Reply:
Will be done in separate PR.